text/0006-shipyard-next-generation.md

```diff
@@ -0,0 +1,317 @@
+# The next generation of Shipyard
+
+The next generation of Shipyard to allow custom delivery/remediation processes.
+
+## Motivation
+
+*There's always room for smart enhancements* - the Shipyard specification is no exception.
+
+A shipyard allows a **Site Reliability Engineer (SRE)** to specify WHAT should happen in the course of an artifact delivery or remediation process. Due to certain use cases, it has been identified that the current specification of the shipyard has limitations and hides details from the SRE. This KEP proposes an enhancement of the *Shipyard* specification to bring flexibility into the definition of the *WHAT*, but still provides an opinionated approach and smart defaults.
+
+Please read this KEP in combination with the KEP: [First version of a Keptn uniform](https://github.com/keptn/enhancement-proposals/pull/7); especially regarding the eventing part.
+
+## Explanation
+
+:man:/:blonde_woman: The target persona of this KEP is an *SRE*, who is responsible for processes and workflows to automate application delivery or remediation operations. 
```

Comment:
> The target persona of this KEP is an *SRE*
change to:
For instance, one target persona of this KEP could be an *SRE (Site Reliability Engineer)*

Reply:
Changed it accordingly. Thx!